packages/i2c/i2c.0.1.1-0/opam

```diff
@@ -18,8 +18,8 @@ build-doc: [ "ocaml" "setup.ml" "-doc" ]
 depends: [
   "result"
   "base-unix"
-  "stdint"
+  "stdint" {<= "0.3.0"}
   "ocamlfind" {>= "1.5"}
   "ocamlbuild" {build}
 ]
-available: [ os = "linux" ]
+available: [ os = "linux" & os != "alpine" ]
```

Comment:
I see; I added a _conf-package_ to address the real dependency (linux kernel header files being available)